heapster.go

```diff
@@ -40,7 +40,7 @@ var (
 	argSinkFrequency   = flag.Duration("sink_frequency", 10*time.Second, "Frequency at which data will be pushed to sinks")
 	argCacheDuration   = flag.Duration("cache_duration", 5*time.Minute, "The total duration of the historical data that will be cached by heapster.")
 	argUseModel        = flag.Bool("use_model", false, "When true, the internal model representation will be used")
-	argModelResolution = flag.Duration("model_resolution", 2*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
+	argModelResolution = flag.Duration("model_resolution", 1*time.Minute, "The resolution of the timeseries stored in the model. Applies only if use_model is true")
```

Comment:
I guess the underlying issue here is that of combining metrics storage and aggregation. We can store container and node metrics at a much higher granularity. But we do not want to aggregate at higher granularities. Update() needs to be broken down. Can you file an issue?